lib/ipmi_quantaoem.c

```diff
@@ -84,12 +84,26 @@
 #define QCT_MAGIC_3 0x00
 #define QCT_MAGIC_4 0x02
 
+struct ipmi_rs* send_cmd(struct ipmi_intf *intf, uint8_t netfn, uint8_t cmd, uint8_t* msg_data, int data_len){
```

Comment:
Function return type must be on a separate line.
All asterisks should stick to the variable names (for function names the asterisks stick to the data type).
Add a space before `{`.
Break the line and align appropriately to fit into 80 positions.

All in all, check https://github.com/ipmitool/ipmitool/wiki/Coding-Standards